modules/monitoring/files/usr/lib/nagios/plugins/check_reboots_required

```diff
@@ -21,12 +21,20 @@ def hostname_with_node_name(hostname, machines_names_hash)
   else
     hostname
   end
+end
 
+def print_hostnames(hostnames, machines_names_hash)
+  hostnames.sort.each do |hostname|
+    hydrated_name = hostname_with_node_name(hostname, machines_names_hash)
+    puts "- #{hydrated_name}"
+  end
 end
 
 all_machines = `/usr/local/bin/govuk_node_list`.split("\n")
 
-unsafe_machines = `/usr/local/bin/govuk_node_list --puppet-class govuk_safe_to_reboot::careful,govuk_safe_to_reboot::no`.split("\n")
+out_of_hours_unsafe_machines = `/usr/local/bin/govuk_node_list --puppet-class govuk_safe_to_reboot::no`.split("\n")
```

Comment:
Minor: the use of "no" contradicts the fact that we're suggesting to reboot it. Appreciate this is a historical issue with the choice of word. It would be clearer if we could rename the strings to "in-hours" and "out-of-hours" or similar.

Reply:
I think I'll leave this for now because I'm unsure what effect that would have, but I'll make a note to see if we can get RE or Platform Health to change it, because I agree the phrasing is confusing.